app/mailers/user_mailer.rb

```diff
@@ -353,11 +353,14 @@ def prompter_notification(work_id, collection_id=nil)
   def delete_work_notification(user, work)
     @user = user
     @work = work
-    work_copy = generate_attachment_content_from_work(work)
-    work_copy = ::Mail::Encodings::Base64.encode(work_copy)
-    filename = work.title.gsub(/[*:?<>|\/\\\"]/,'')
-    attachments["#{filename}.txt"] = { content: work_copy, encoding: "base64" }
-    attachments["#{filename}.html"] = { content: work_copy, encoding: "base64" }
+    download = Download.new(@work, mime_type: "text/html")
+    html = DownloadWriter.new(download).generate_html
+    txt = ActionController::Base.helpers.strip_tags(html)
```

Comment:
I just noticed this after reading old comments on the issue: would this remove all HTML tags in the output, including user-created ones in the work body?

We can keep doing what we're doing now, which is the same output for both types of attachments.

Reply:
Yes, the previous behaviour was to attach the same file twice but with different filenames, which didn't seem particularly useful to me.  Do we want to go back to that behaviour?  I feel like this gives you the option of a true plain text version, versus HTML just saved as text (which you technically have anyway, if you rename the HTML file when you download it).

Do we want to put it back to the way it was before?

Reply:
Let's put it back to the way it was.

I don't think users would want a version where their formatting gets removed.

Having the plain text file with the same content makes sure that people have a way to open and view works with the HTML visible (easy to repost elsewhere). Some devices may only display the HTML file as rich text, or [have problems opening it at all](https://support.google.com/chrome/thread/32010024/cant-open-local-html-file-on-chrome?hl=en).